virttest/element_tree.py

```diff
@@ -852,7 +852,7 @@ def fixtag(tag, namespaces):
     # tag and namespace declaration, if any
```

Comment:
```
import string


try:
    REPLACE = string.replace
except AttributeError:
    REPLACE = str.replace
```

we can remove this `string` import aswell and make `REPLACE=str.replace`

Reply:
ok, thanks, I will remove them.